manpage/wf-recorder.1.scd

```diff
@@ -20,7 +20,30 @@ wf-recorder - A simple screen recording program for wlroots-based compositors
 *-c, --codec*
 	Specifies the codec of the video. Supports GIF output as well.
 
-	To modify codec parameters, use *-p <option_name>=<option_value>*
+*-r, --framerate*
```

Comment:
I think you should stress that this is just an approximation and doesn't really change the refresh rate, or does it?

Reply:
It's been a while since I made this, but IIRC this was because the actual frame rate sometimes isn't actually achieved? Perhaps it would be better described as target frame rate. It attempts to reach it, but if the encoding is not fast enough, in practice it will be a lower frame rate. 

It should also be noted that when recording at a high frame rate, and also having heavy encoding, there can be desyncing issues if it can't actually write frames fast enough. I am not currently sure what is needed to fix this, other than by using easier encoding and recording at a lower framerate. If Wayland is anything like X11, it is also possible that actually retrieving the frames could be a bottleneck.